### PR TITLE
[Partner Dashboard] - Credit Transfer

### PR DESCRIPTION
**Pull request by guganabu**

### CODE REVIEW

File: app/components/partner/credit-transfer/index.scss [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.back-btn {
+  border-color: rgba(#424651, 0.7);
+  color: rgba(#424651, 0.7);
+  &:hover {
+    border-color: rgba(#424651, 1);
+    color: rgba(#424651, 1);
+  }
+}
```
Comment (Praseetha-KR) [inline app/components/partner/credit-transfer/index.scss:121]: Don't introduce new buttons, use the existing ones

File: app/components/partner/credit-transfer/index.scss [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  row-gap: 1.5rem;
+}
+
+.transferable-credits {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #0063b2ff;
```
Comment (Praseetha-KR) [inline app/components/partner/credit-transfer/index.scss:51]: Specifying colors randomly in component level has become unmaintainable. 

Let's restrain from not directly specifying any colors in components via css module classes, instead use color util classes.

**Review by Praseetha-KR — CHANGES_REQUESTED** [review]

File: app/components/partner/client-info/template.hbs [hunk 3]
```diff
@@ -42,6 +42,12 @@
               data-test-payment-plan
               @clientId={{@client.id}}
             />
+            {{#if this.partner.access.transfer_credits}}
+              <Partner::CreditTransfer
+                @client={{@client}}
+                data-test='credit-transfer'
+              />
+            {{/if}}
```
[the remark this answers is not in the document]
Comment (guganabu) [reply app/components/partner/client-info/template.hbs:50]: Yeah, checked the commented container and moved the button inside the credit-transfer component 👍

File: app/components/partner/client-info/template.hbs [hunk 4]
```diff
@@ -42,6 +42,12 @@
               data-test-payment-plan
               @clientId={{@client.id}}
             />
+            {{#if this.partner.access.transfer_credits}}
+              <Partner::CreditTransfer
+                @client={{@client}}
+                data-test='credit-transfer'
```
[the remark this answers is not in the document]
Comment (guganabu) [reply app/components/partner/client-info/template.hbs:48]: Got to know about the ember test selectors, will change it to plain attribute

File: translations/en.json [hunk 5]
```diff
@@ -839,5 +839,20 @@
...
+  "availableCredits": "Available Scan Credits",
+  "creditAddPurchase": "Please contact support for purchasing scan credits",
+  "creditTransferNotSupported": "Scan credit sharing is currently not supported in your payment plan (per-app). Please contact support for transferring credits to clients.",
+  "remainingCredits": "Remaining Credits",
+  "client": "Client",
+  "currentCredits": "Current Credits",
+  "newCredits": "New Credits",
+  "confirmTransfer": "Confirm Transfer",
+  "creditTransferSuccess": "Credits transferred to the client successfully",
+  "creditTransferError": "Couldn't transfer credits, please try again later!"
```
[the remark this answers is not in the document]
Comment (guganabu) [reply translations/en.json:857]: I have sorted the whole en.json & ja.json in alphabetical order by key

File: app/components/partner/credit-transfer/index.scss (hunk 1 above)
Comment (guganabu) [reply app/components/partner/credit-transfer/index.scss:121]: Yeah, saw the is-default class and used the same for back button

File: app/components/partner/credit-transfer/index.scss (hunk 2 above)
Comment (guganabu) [reply app/components/partner/credit-transfer/index.scss:51]: Agree, we may have to define color codes as variables, may be start adding while upgrading the ux.

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**